lib/tasks/refinery/elasticsearch.rake

```diff
@@ -15,6 +15,9 @@ namespace :refinery do
 
     desc 'Recreate Elasticsearch index'
     task recreate: :setup_logger do
+      # Use the right language
+      ::I18n.locale = ::I18n.default_locale
```

Comment:
For my multilang project i use this config : 

```ruby
# app/decorators/models/refinery

Refinery::Page.class_eval do
  include ::Refinery::Elasticsearch::Searchable

  define_mapping do
    {
      title_fr: { type:'string', boost:2.0 },
      title_en: { type:'string', boost:2.0 },
      browser_title: { type:'string' },
      menu_title_fr: { type:'string' },
      menu_title_en: { type:'string' },
      meta_description: { type:'string' },
      part: {
        type:'object',
        properties: {
          body_fr: { type:'string', analyzer:'snowball' },
          body_en: { type:'string', analyzer:'snowball' }
        }
      },
      created_at: { type:'date' },
      updated_at: { type:'date' }
    }
  end

  def to_index
    if in_menu?
      {
        title_fr: self.title_fr,
        title_en: self.title_en,
        part: self.parts.map{ |part|
          {
            body_fr: strip_tags(part.body_fr),
            body_en: strip_tags(part.body_en)
          }
        },
        browser_title: self.browser_title,
        menu_title_fr: self.menu_title_fr,
        menu_title_en: self.menu_title_en,
        meta_description: self.meta_description,
        created_at: self.created_at,
        updated_at: self.updated_at
      }
    end
  end

  def self.indexable
    live.in_menu
  end
end
```

i've no changes in the rake task.